### DISCUSSION TITLE
April 2023 Melbourne MicroPython Meetup

### DISCUSSION THREAD

**Discussion by mattytrentini** [micropython/micropython#11331] [General]
The [April Melbourne MicroPython Meetup](https://www.meetup.com/micropython-meetup/events/292404130/) is on tomorrow! That's Wednesday evening in Melbourne, but you can see the [time in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?msg=April+Melbourne+MicroPython+Meetup&iso=20230426T1830&p1=152&ah=3).

It'll be held in-person but will be [live-streamed on zoom](https://planetinnovation.zoom.us/j/94632747010?pwd=Q1dBRHM3MzhSa0hOVjZ1RlFVR2g0Zz09) and recorded. 

Hope to see you there, or on the stream!

**Comment by mattytrentini** [comment]
[News Roundup Slides](https://docs.google.com/presentation/d/e/2PACX-1vS7oU-US5MN1bXTbPG4ZNFtVJuOYQx5GOdIOURQVP7ESSm08sz7M9q_vFXxCaihmjng2Sl2KwbtcGbQ/pub) are now available.
Damien discussed the hot-off-the-press [v1.20.0](https://github.com/orgs/micropython/discussions/11342).

Videos coming soon!

Also, during the evening, Oliver published the [m5-8encoder](https://github.com/HowManyOliversAreThere/m5-8encoder) driver for the [M5Stack 8Encoder](https://docs.m5stack.com/en/unit/8Encoder). @andrewleech worked on a very cool looking [asyncio software-only profiler](https://github.com/micropython/micropython-lib/pull/650) (I know @peterhinch will be interested!).

**Comment by sosi-deadeye** [comment]
Btw. please change the thumbnail of this Video: https://www.youtube.com/watch?v=FdZj5zUj5UE

You hit him very unlucky.

**Reply by mattytrentini** [reply]
Updated, thanks!
